lua/oil/init.lua

```diff
@@ -621,6 +624,12 @@ M.select = function(opts, callback)
         args = { filebufnr },
         mods = mods,
       })
+
+      if not opts.preview and preview_win and entry_is_file then
+        vim.api.nvim_win_close(preview_win, true)
+        vim.api.nvim_set_current_win(prev_win)
```

Comment:
Is this line necessary? Is it possible that the cursor has changed windows between line 533 and here?

Reply:
Good point, yeah I think we can remove that line — the only case where the current window changes is if `opts.preview`